guides/source/action_controller_overview.md

```diff
@@ -158,13 +178,21 @@ So for example, if you are sending this JSON content:
 
 Your controller will receive `params[:company]` as `{ "name" => "acme", "address" => "123 Carrot Street" }`.
```

Comment:
We used `params[:client]` in the example above but here it changes to `params[:company]` which could potentially be confusing.

Reply:
Hm...they are two separate subsections and examples. Nonetheless, changed both to `user` as "client" is not common and can be confused with browser client or something.